ledger-service/http-json/src/itlib/scala/http/AbstractDatabaseIntegrationTest.scala

```diff
@@ -39,8 +38,7 @@ abstract class AbstractDatabaseIntegrationTest extends AsyncFreeSpecLike with Be
 
   "DbStartupOps" - {
 
-    implicit lazy val logHandler: log.LogHandler = dao.logHandler
-    implicit lazy val jdbcDriver: SupportedJdbcDriver = dao.jdbcDriver
+    import dao.{logHandler, jdbcDriver}, jdbcDriver.q.queries
 
```

Comment:
Not only does it work, I _strongly recommend_ that you always do this instead of defining a fresh binding, whenever possible.

Morally speaking, in terms of Scala identifier semantics, it is no different from doing `import scalaz.std.option._` to get the monad instance rather than defining a fresh `implicit val blahblah: Monad[Option] = ...` every time you want to use it. **`option` is a value.**

Reply:
Thank you for the insight. That is useful!